### ISSUE TITLE
Guest Email Bug

### ISSUE THREAD

**Issue by jmwolff3** [reactioncommerce/reaction#959]
### Expected behavior
```
INFO Reaction: setting default mail url to: smtp.gmail.com
INFO Reaction: orders/sendNotification coreOrderWorkflow/processing
INFO Reaction: Unable to send notification email: Error: getaddrinfo ENOTFOUND
```

The expected behavior is the email fails to send, but the process of marking the shipment as completed continues.

### Actual Behavior
```
INFO Reaction: setting default mail url to: smtp.gmail.com
INFO Reaction: orders/sendNotification coreOrderWorkflow/processing
FATAL Reaction: Unable to send notification email: Error: getaddrinfo ENOTFOUND
```

The order can not be marked as completed.

### Steps to Reproduce the Behavior

Checkout as a guest, but add an email address for updates.  When marking the item as shipped, the FATAL reaction occurs and the order can not be marked as complete.

**Comment by zenweasel** [comment]
This is similar to the issue we had with checkout completion where waiting for the email to send was causing the completed screen to take a long time. We just need to make sending email a non-critical path function.

**Comment by aaronjudd** [comment]
Related to #1061

**Comment by zenweasel** [comment]
Also related to https://github.com/reactioncommerce/reaction/issues/1200 I believe

**Comment by aaronjudd** [comment]
Resolved in #1282 - I was unable to replicate issue in testing that PR.
